src/components/Search/Search.js

```diff
@@ -0,0 +1,27 @@
+import { MDCTextField } from "@material/textfield";
+
+export class SearchComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  querySelectors() {
+    this.textField = new MDCTextField(
+      document.querySelector(".mdc-text-field")
+    );
+  }
+
+  mount() {
+    this.mountPoint.innerHTML = this.render();
+    this.querySelectors();
+  }
+
+  render() {
+    return `
+      <div class="mdc-text-field mdc-text-field--outlined mdc-text-field--with-leading-icon">
+        <i class="material-icons mdc-text-field__icon">search</i>
```

Comment:
could you please use text field from here https://material-components.github.io/material-components-web-catalog/#/component/text-field. There is a rounded one too among the demos